### PR TITLE
Do not convert IShape to dynamic

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.Fields.BulkActions.cshtml [hunk 1]
```diff
@@ -7,8 +7,11 @@
     </button>
     <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="bulk-action-menu-button">
         @{
-            dynamic bulkActions = await ContentOptionsDisplayManager.BuildDisplayAsync(Model, null, "BulkActions");
-            @await DisplayAsync(bulkActions.Content)
+            var bulkActions = await ContentOptionsDisplayManager.BuildDisplayAsync(Model, null, "BulkActions");
+            if (bulkActions.TryGetProperty<IShape>("Content", out var content))
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.Fields.BulkActions.cshtml:11]: Do we have to call "Try" here?

**Comment by sebastienros** [comment]
Can you also change `ValueTask<IShape> PagerAsync()` in `ShapeFactoryExtensions.cs` ?

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.Fields.BulkActions.cshtml (hunk 1 above)
Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.Fields.BulkActions.cshtml:11]: We can call `Property<IShape>()` but what if it does not exists?

**Comment by MikeAlhayek** [comment]
> Can you also change ValueTask<IShape> PagerAsync() in ShapeFactoryExtensions.cs ?

@sebastienros ShapeFactoryExtensions was updated and it's already in main.

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.Fields.BulkActions.cshtml (hunk 1 above)
Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.Fields.BulkActions.cshtml:11]: OK, let's assume the Content zone may not be created after BuilddisplayAsync.

**Review by sebastienros — APPROVED** [review]

**Pull request merged**